components/create_post/create_post.test.jsx

```diff
@@ -27,7 +27,7 @@ jest.mock('react-dom', () => ({
     }),
 }));
 
-jest.mock('actions/post_actions.jsx', () => ({
+jest.mock('actions/post_actions.jsx', (process) => ({
```

Comment:
This existing code was causing the hoist checker to fail.

Reply:
Yes, but I think your proposal is not correct, the jest.mock second parameter is a factory that receive no parameters and returns the mock, so... that way process should be undefined, not the "process" as expected.